### PR TITLE
Install binaries

### PR DESCRIPTION
**Pull request by banasiakadam60**

### CODE REVIEW

File: Installer/Installer_Win/install_script.iss [hunk 1]
```diff
@@ -13,7 +13,7 @@
 ; https://download.docker.com/win/stable/InstallDocker.msi
 ; https://sourceforge.net/projects/pywin32/files/pywin32/Build%20220/pywin32-220.win32-py2.7.exe/download
 #define Repository "C:\golem"           
-#expr Exec("powershell.exe python setup.py bdist_wheel", "", Repository, 1)
+;#expr Exec("powershell.exe python setup.py pyinstaller", "", Repository, 1)
```
Comment (mfranciszkiewicz) [inline Installer/Installer_Win/install_script.iss]: Please uncomment

File: Installer/Installer_Win/install_script.iss [hunk 2]
```diff
@@ -75,9 +75,9 @@ Source: "{#Repository}\Installer\Installer_Win\deps\OpenSSL\ssleay32.dll"; DestD
...
 [Icons]
-Name: "{commonprograms}\{#MyAppName}"; Filename: "{sd}\Python27\Scripts\golemapp.exe"; IconFilename: "{app}\{#AppIcon}"
-Name: "{commondesktop}\{#MyAppName}"; Filename: "{sd}\Python27\Scripts\golemapp.exe"; IconFilename: "{app}\{#AppIcon}"; Tasks: desktopicon
-Name: "{userappdata}\Microsoft\Internet Explorer\Quick Launch\{#MyAppName}"; Filename: "{sd}\Python27\Scripts\golemapp.exe"; IconFilename: "{app}\{#AppIcon}"; Tasks: quicklaunchicon
+Name: "{commonprograms}\{#MyAppName}"; Filename: "{app}\golemapp.exe"; IconFilename: "{app}\{#AppIcon}"
+Name: "{commondesktop}\{#MyAppName}"; Filename: "{app}\golemapp.exe"; IconFilename: "{app}\{#AppIcon}"; Tasks: desktopicon
+Name: "{userappdata}\Microsoft\Internet Explorer\Quick Launch\{#MyAppName}"; Filename: "{app}\golemapp.exe"; IconFilename: "{app}\{#AppIcon}"; Tasks: quicklaunchicon
 
 [Run]
  ; Install Python 2.7.6
```
Comment (mfranciszkiewicz) [inline Installer/Installer_Win/install_script.iss]: Not needed

File: Installer/Installer_Win/install_script.iss [hunk 3]
```diff
@@ -103,9 +103,6 @@ Filename: "{tmp}\DockerToolbox.exe"; Parameters: "/SILENT"; StatusMsg: "Installi
 ; Install geth
 Filename: "{tmp}\geth-windows-amd64-1.5.9-a07539fb.exe"; StatusMsg: "Installing geth"; Description: "Install geth"
 
-; Finally! Install golem!
-Filename: "powershell.exe"; Parameters: "-Command ""{sd}\Python27\Scripts\pip.exe install """"{app}\golem-{#SetupSetting("AppVersion")}-cp27-none-win32.whl"""""""; Description: "Install Golem"; Check: DependenciesSetup('Golem')
-
 ; Install pywin
 Filename: "{tmp}\pywin32-220.win32-py2.7.exe"; StatusMsg: "Installing pywin32"; Description: "Install pywin32"
```
Comment (mfranciszkiewicz) [inline Installer/Installer_Win/install_script.iss]: Not needed

**Review by mfranciszkiewicz — COMMENTED** [review]
`hyperg` is missing from `PATH`

File: Installer/Installer_Win/install_script.iss (hunk 3 above)
Comment (banasiakadam60) [reply Installer/Installer_Win/install_script.iss]: Done

File: Installer/Installer_Win/install_script.iss (hunk 2 above)
Comment (banasiakadam60) [reply Installer/Installer_Win/install_script.iss]: Done

File: Installer/Installer_Win/install_script.iss (hunk 1 above)
Comment (banasiakadam60) [reply Installer/Installer_Win/install_script.iss]: Done

File: Installer/Installer_Linux/install.sh [hunk 4]
```diff
@@ -17,6 +17,7 @@ declare -r docker_script='docker_install.sh'
 declare -r version_file='version'
 declare -r ipfs_url='https://dist.ipfs.io/go-ipfs/v0.4.6/'
 declare -r ipfs_package='go-ipfs_v0.4.6_linux-amd64.tar.gz'
+declare -r hyperg='https://github.com/mfranciszkiewicz/golem-hyperdrive/releases/download/v0.1.2/hyperg_0.1.2_linux-amd64.tar.bz2'
```
Comment (mfranciszkiewicz) [inline Installer/Installer_Linux/install.sh:20]: Please link `hyperg` to (e.g.) `/usr/local/bin`

File: Installer/Installer_Win/install_script.iss [hunk 5]
```diff
@@ -42,9 +41,6 @@ SolidCompression=yes
 ; Set environment variable to point to company installation
 Root: "HKLM64"; Subkey: "SYSTEM\CurrentControlSet\Control\Session Manager\Environment"; ValueType: string; ValueName: "GOLEM"; ValueData: "{sd}\Python27\Scripts\golemapp.exe"; Flags: uninsdeletevalue;
```
Comment (mfranciszkiewicz) [inline Installer/Installer_Win/install_script.iss]: Please update.

File: Installer/Installer_Win/install_script.iss [hunk 6]
```diff
@@ -42,9 +41,6 @@ SolidCompression=yes
...
  
-; Append python to PATH if does not already exist
-Root: "HKLM64"; Subkey: "SYSTEM\CurrentControlSet\Control\Session Manager\Environment"; ValueType: expandsz; ValueName: "PATH"; ValueData: "{olddata};{sd}\Python27\;{sd}\Python27\Scripts\"; Check: NeedsAddPath('{sd}\Python27')
-
 ; Append Docker to PATH
```
Comment (mfranciszkiewicz) [inline Installer/Installer_Win/install_script.iss]: Please also install and add `hyperg` to `PATH`

**Review by mfranciszkiewicz — DISMISSED** [review]
- remove `python-dev python-pyqt5 pyqt5-dev-tools python-netifaces python-psutil build-essential python-pip` (as in python-related and build-related packages) from the linux installer
- remove python package installation via pip from the linux installer

File: Installer/Installer_Linux/install.sh (hunk 4 above)
Comment (banasiakadam60) [reply Installer/Installer_Linux/install.sh:20]: Done

File: Installer/Installer_Win/install_script.iss (hunk 5 above)
Comment (banasiakadam60) [reply Installer/Installer_Win/install_script.iss]: Done

File: Installer/Installer_Win/install_script.iss (hunk 6 above)
Comment (banasiakadam60) [reply Installer/Installer_Win/install_script.iss]: Done

File: Installer/Installer_Linux/install.sh [hunk 7]
```diff
@@ -94,9 +96,7 @@ function install_dependencies()
 {
     info_msg "INSTALLING GOLEM DEPENDENCIES"
     apt-get update
-    apt-get install -y openssl python-dev python-pyqt5 qt5-default pyqt5-dev-tools libffi-dev pkg-config libjpeg-dev libopenexr-dev libssl-dev autoconf libgmp-dev libtool python-netifaces python-psutil build-essential python-pip
-    pip install --upgrade pip
-    pip install service-identity zope.interface websocket-client openexr certifi devp2p cbor2 dill base58 multihash ovh weakreflist
+    apt-get install -y openssl qt5-defaultlibffi-dev pkg-config libjpeg-dev libopenexr-dev libssl-dev autoconf libgmp-dev libtool
```
Comment (mfranciszkiewicz) [inline Installer/Installer_Linux/install.sh]: `qt5-defaultlibffi-dev`

Comment (banasiakadam60) [reply Installer/Installer_Linux/install.sh]: Done

Comment (mfranciszkiewicz) [reply Installer/Installer_Linux/install.sh]: we still need `qt5-default`, package names were concatenated

**Pull request merged**